akka-remote/src/main/java/akka/remote/artery/AeronErrorLog.java

```diff
@@ -74,8 +71,8 @@ public long logErrors(LoggingAdapter log, long sinceTimestamp)
                   log.error(String.format(
                       "Aeron error: %d observations from %s to %s for:%n %s",
                       observationCount,
-                      formatter.format(LocalDateTime.ofInstant(Instant.ofEpochMilli(firstObservationTimestamp), timeZone)),
-                      formatter.format(LocalDateTime.ofInstant(Instant.ofEpochMilli(lastObservationTimestamp), timeZone)),
+                      Helpers.timestamp(firstObservationTimestamp),
+                      Helpers.timestamp(lastObservationTimestamp),
```

Comment:
yes, I prefer the ISO format, but we can't change and better to use the same everywhere